examples/js/controls/TrackballControls.js

```diff
@@ -687,6 +687,16 @@
 
 						break;
 
+					case 2:
```

Comment:
Please always edit files in the `examples/jsm` directory. `examples/js` is generated.

Reply:
Thanks for the reminder !
Should I submit another pr ?

Reply:
Not necessary.